File: CodeForces/1701-1800/1760/A/1760A.cs
// Problem 1760A: Medium Number
// https://codeforces.com/contest/1760/problem/A
// Compiled with: Mono JIT compiler version 6.8.0.105 (Debian 6.8.0.105+dfsg-3.3+deb12u1 Sat Jun 21 16:33:59 UTC 2025)
// Linux version: Linux kernel 6.1.0-10-amd64
// Submitted on: October 23rd, 2025
// Codeforces language used: C# Mono 6.8

using System;

class Program {
    static void Main() {
        int n = int.Parse(Console.ReadLine());
        for (int i = 0; i < n; i++) {
            string all_input = Console.ReadLine();
            string[] parts = all_input.Split(' ');
            int a = int.Parse(parts[0]);
            int b = int.Parse(parts[1]);
            int c = int.Parse(parts[2]);
            
            if (a < b){
                // Possible cases:
                // 1-2-3, 1-3-2, 2-3-1
                if (a < c){
                    //Possible cases:
                    // 1-2-3, 1-3-2
                    Console.WriteLine(Math.Min(b, c));
                } else {
                    //Possible cases:
                    // 2-3-1
                    Console.WriteLine(a);
                }
            } else {
                // Possible cases:
                // 2-1-3, 3-1-2, 3-2-1
                if (a > c) {
                    // Possible cases:
                    // 3-1-2, 3-2-1
                    Console.WriteLine(Math.Max(b,c));
                } else {
                    // Possible cases:
                    // 2-1-3
                    Console.WriteLine(a);
                }
            }

        }
    }
}

File: CodeForces/1701-1800/1760/A/1760A.rs
// Problem 1760A: Medium Number
// https://codeforces.com/contest/1760/problem/A
// Compiled with: rustc 1.85.1 (4eb161250 2025-03-15) (built from a source tarball)
// Linux version: Linux kernel 6.17.0-6-generic
// Submitted on: October 27th, 2025
// Codeforces language used: Rust 1.89.0 (2024)

fn main() {
    let runs: i32 = std::io::stdin().lines().next().unwrap().unwrap().trim().parse().unwrap();

    for _ in 0..runs {
        let mut nums: Vec<i32> = std::io::stdin().lines().next().unwrap().unwrap().split_whitespace().map(|x| x.parse::<i32>().unwrap()).collect();
        nums.sort_by_key(|x| *x);
        println!("{}", nums[1]);
    }
}
